dashboard/client/pages/index.tsx

```diff
@@ -1,8 +1,43 @@
-import { NextPage } from 'next'
+import { NextStatelessComponent } from 'next'
 import React from 'react'
 
-const HomePage: NextPage = () => {
-  return <div>Welcome to Next.js!</div>
+import Ticket from '../components/tickets'
+
+import axios from 'axios'
+import Layout from '../components/layout'
+
+import { getSpaces, getTickets } from '../gql/query'
+
+interface IProps {
+  spaces: any[]
+  tickets: any[]
+}
+
+const HomePage: NextStatelessComponent<IProps> = props => {
+  return (
+    <Layout spaces={props.spaces}>
+      {props.tickets.map(x => (
```

Comment:
Don't forget to check if `tickets` has a value!

```js
const HomePage: NextStatelessComponent<IProps> = ({ spaces, tickets }) => (
  <Layout spaces={spaces}>
    {tickets && tickets.map(x => <Ticket key={x.title} ticket={x} />)}
  </Layout>
)
```

Reply:
what do u men

Reply:
we check if `tickets` is truthy first w/ `&&`, if we don't, JS will throw an error sayin' can't find `.map` on `undefined`

Reply:
lol i thought that code snippet that you pasted was my code and i was like ... i am doing that already lol

Reply:
lmao 😆